functions/Add-DbaComputerCertificate.ps1

```diff
@@ -78,36 +75,39 @@ Adds the local C:\temp\cer.cer to the local computer's LocalMachine\My (Personal
 			return
 		}
 		
-		if (!(Test-Path -Path $path)) {
-			Write-Message -Level Warning -Message "$Path does not exist"
-			return
+		if ($Path) {
+			if (!(Test-Path -Path $Path)) {
+				Write-Message -Level Warning -Message "$Path does not exist"
+				return
+			}
```

Comment:
Input Validations of non-piped values should be part of `begin`
Use Stop-Function to interrupt due to bad parameterization